ios_tests/appium.txt

```diff
@@ -1,6 +1,6 @@
 [caps]
 platformName = "ios"
-platformVersion = "9.2"
+platformVersion = "8.3"
```

Comment:
Update `platformVersion`s and related documentations in https://github.com/appium/ruby_lib/pull/389/commits/f93252ebdb4fc699139d75ac86378c8002ff2665